### PR TITLE
Feature/initial release

### PR DESCRIPTION
**Pull request by tomme87**
Initial version

1. Finds the latest tag defined on HEAD
2. Find all JIRA issue keys in the tag message
3. Create a release with the tag as name and sets fixVersion on all the issues found to this version

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -15,10 +15,3 @@ jobs:
           npm install
       - run: |
           npm run all
-  test: # make sure the action works on a clean machine without building
```
Comment (erichjsonfosse) [inline .github/workflows/test.yml:18]: Name of the file could be 'build-test', no?

File: README.md [hunk 2]
```diff
@@ -1,105 +1,40 @@
...
 
-Use this template to bootstrap the creation of a TypeScript action.:rocket:
+Creates a JIRA release based on a git tag.
 
-This template includes compilation support, tests, a validation workflow, publishing, and versioning guidance.  
+# How it works:
 
-If you are new, there's also a simpler introduction.  See the [Hello World JavaScript Action](https://github.com/actions/hello-world-javascript-action)
+1. Finds the latest tag defined on HEAD
+2. Find all JIRA issue keys in the tag message
```
Comment (erichjsonfosse) [inline README.md:12]: Typo: 'Finds'

File: README.md [hunk 3]
```diff
@@ -1,105 +1,40 @@
...
+3. Create a release with the tag as name and sets fixVersion on all the issues found to this version
```
Comment (erichjsonfosse) [inline README.md:13]: Typo: 'Creates'

File: src/jira.ts [hunk 4]
```diff
@@ -0,0 +1,77 @@
+import * as core from '@actions/core'
+import JiraApi from 'jira-client'
+
+export class Jira extends JiraApi {
+  private jiraIdRegex = /((?<!([A-Za-z]{1,10})-?)[A-Z]+-\d+)/g
```
Comment (erichjsonfosse) [inline src/jira.ts:5]: What's with the negative lookbehind?

**Review by erichjsonfosse — CHANGES_REQUESTED** [review]

File: .github/workflows/test.yml (hunk 1 above)
Comment (tomme87) [reply .github/workflows/test.yml:18]: Renamed the filed to build.yml, since we only build here now.

File: src/jira.ts (hunk 4 above)
Comment (tomme87) [reply src/jira.ts:5]: It's taken from https://stackoverflow.com/a/30518972
It prevents something like XX-JIRA-123 and abcJIRA-123 from matching.

**Review by erichjsonfosse — APPROVED** [review]

**Pull request merged**